packages/flutter/lib/src/rendering/mouse_tracking.dart

```diff
@@ -332,10 +332,17 @@ abstract class BaseMouseTracker extends ChangeNotifier {
   LinkedHashMap<MouseTrackerAnnotation, Matrix4> _hitTestResultToAnnotations(HitTestResult result) {
     assert(result != null);
     final LinkedHashMap<MouseTrackerAnnotation, Matrix4> annotations = <MouseTrackerAnnotation, Matrix4>{}
-        as LinkedHashMap<MouseTrackerAnnotation, Matrix4>;
+      as LinkedHashMap<MouseTrackerAnnotation, Matrix4>;
     for (final HitTestEntry entry in result.path) {
-      if (entry.target is MouseTrackerAnnotation) {
-        annotations[entry.target as MouseTrackerAnnotation] = entry.transform!;
+      final HitTestTarget target = entry.target;
+      if (target is MouseTrackerAnnotation) {
+        if (target is RenderObject) {
```

Comment:
It is odd that we have to special case `RenderObject`s here. What about other MouseTrackerAnnotations that have become "invalid"?

Reply:
Do we have a situation where the MouseTrackerAnnotations is not renderObject?

Reply:
We're about to. https://github.com/flutter/flutter/pull/68674 is introducing a subclass of `TextSpan` to be a `MouseTrackerAnnotations`. Although there is no way for that to be turned off. I'm thinking if we should have a generalized way of turning it off.

Reply:
If we introduce a new property of MouseTrackerAnnotation to indicator whether it is valid, does the Span have chance to set it?

Reply:
@dkwingsmt Hi, Sorry to bother you again ：）
WDYT of a generalized way of turning it off without breaking any API?